### PR TITLE
Add first docstrings

### PR DESCRIPTION
**Pull request by yeyeto2788**
Hello guys! this first commit is to see if this way of working is approved for you.

I have created a branch called `docstring` which can be also added on the repo so all changes can be merge into this branch and then merge it into master.

I also wanted to point out that there are some comment on the `App.py` that state that those methods are deprecated, I didn't add docstring in there because of that.

I'll keep updating the repo after the files are revised.

Best regards.

P.D: This PR is related to issue #44

### CODE REVIEW

**Comment by yeyeto2788** [comment]
Hey @lawsie thank you for replying, for me it is a pleasure to contribute on the project.

I'll wait for @martinohanlon response so I can proceed further.

Regards.

**Comment by martinohanlon** [comment]
Thanks a lot for your help :) I will provide comments as a review of the PR, dont be surprised if it seems a bit "picky", its not my intention (I can just come across that way!).

File: guizero/alerts.py [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
+
 
 def error(title, text):
+    """
+    Alert type error
+
+    :param string title:
+        String with the title to be apply.
+    :param string text:
+        String with the text to be displayed.
```
Comment (martinohanlon) [inline guizero/alerts.py]: \- String with

File: guizero/alerts.py [hunk 2]
```diff
@@ -1,14 +1,58 @@
...
+    """
+    messagebox.showinfo(title, text)
+
 
 def error(title, text):
+    """
+    Alert type error
+
+    :param string title:
+        String with the title to be apply.
```
Comment (martinohanlon) [inline guizero/alerts.py]: The title to be displayed on the box.

File: guizero/alerts.py [hunk 3]
```diff
@@ -1,14 +1,58 @@
...
+        String with the text to be displayed.
+    :return:
+        Nothing.
+    """
+    messagebox.showinfo(title, text)
+
 
 def error(title, text):
+    """
+    Alert type error
```
Comment (martinohanlon) [inline guizero/alerts.py]: Display an error message box.

File: guizero/alerts.py [hunk 4]
```diff
@@ -1,14 +1,58 @@
...
+    """
     messagebox.showerror(title, text)
 
+
 def yesno(title, text):
+    """
+    Alert type yes or no
+
+    :param string title:
+        String with the title to be apply.
```
Comment (martinohanlon) [inline guizero/alerts.py]: The title to be displayed on the box.

File: guizero/alerts.py [hunk 5]
```diff
@@ -1,14 +1,58 @@
...
+        String with the text to be displayed.
+    :return:
+        Nothing.
+    """
     messagebox.showerror(title, text)
 
+
 def yesno(title, text):
+    """
+    Alert type yes or no
```
Comment (martinohanlon) [inline guizero/alerts.py]: Perhaps:

"""
Display a yes / no message box.

Returns `True` if the "yes" selected, `False` if "no".
"""

File: guizero/alerts.py [hunk 6]
```diff
@@ -1,14 +1,58 @@
...
+
+    :param string title:
+        String with the title to be apply.
+    :param string text:
+        String with the text to be displayed.
```
Comment (martinohanlon) [inline guizero/alerts.py]: \- String with

File: guizero/alerts.py [hunk 7]
```diff
@@ -1,14 +1,58 @@
...
+    :return:
+        Nothing.
```
Comment (martinohanlon) [inline guizero/alerts.py]: :return bool:
    `True` if the "yes" selected, `False` if "no"

**Comment by yeyeto2788** [comment]
Hello @martinohanlon, 

It is ok for me, I was expecting everything to be commented since this PR was essentially to see what the are the, let's say, requirements for adding the docstrings. :smiley:

I'll take a look at the comments and I add more doc strings on other modules as well.

Regards.

**Comment by martinohanlon** [comment]
Thank you, I am planning release `0.6` at the moment, so will merge into this.

**Pull request merged**